### PR TITLE
Updated DATA ID to use VWAP api endpoints directly and no local aggregation

### PR DESCRIPTION
**Pull request by krasi-georgiev**
Signed-off-by: Krasi Georgiev <8903888+krasi-georgiev@users.noreply.github.com>

### CODE REVIEW

**Review by themandalore — APPROVED** [review]

**Comment by imaginebeingatcomputers** [comment]
Looks good. These 2 feeds should bring the tellor feed much closer to Ampleforth's oracle. 

I did some analysis over prices reported by Ampleforth's Oracle, Chainlink's Feed, and the median of these 2 APIs:
This analysis used the oracle data from the ProviderReportPushed event of the median [contract](https://etherscan.io/address/0x99C9775E076FDF99388C029550155032Ba2d8914) and filtered for [Chainlink's](https://etherscan.io/address/0xfc4b1ce32ed7310028dcc0d94c7b3d96dcd880e0) and [Ampleforth's](https://etherscan.io/address/0x8844dfd05ac492d1924ad27ddd5e690b8e72d694) provider addresses.

This Data was pulled from days between 12:00AM GMT May 13, 2021 and 12:00AM GMT August 10, 2021
Average difference of Chainlink and Ampleforth: 0.008682048133942225
Average difference of Anyblock/BNC and Ampleforth: 0.008022412238088333
Average difference of Anyblock/BNC and Chainlink: 0.0007960050570059132

The points plotted alongside each other:
image

I'll also post the script used to generate this data after I clean it of API keys.

File: configs/index.json [hunk 1]
```diff
@@ -95,6 +95,20 @@
...
+    "AMPL/USD/VWAP": {
+        "interval": "10m",
+        "endpoints": [
+            {
+                "Type": "bravenewcoin",
+                "URL": "https://bravenewcoin.p.rapidapi.com/ohlcv?size=1&indexId=551cdbbe-2a97-4af8-b6bc-3254210ed021&indexType=GWA&rapidapi-key=$RAPID_KEY",
+                "param": "$.content[0][vwap]"
+            },
+            {
+                "URL": "https://api.anyblock.tools/market/AMPL_USD_via_ALL/daily-volume?roundDay=false&debug=false&access_token=$ANYBLOCK_KEY&start=$NOW&end=$EOD",
```
Comment (imaginebeingatcomputers) [inline configs/index.json:107]: The AnyBlock API accepts `start` as the timestamp which to begin calculation of VWAP, and `end` for the end of the calculation period.

Swapping $EOD and $NOW should fix the 404 error resulting from their current orientation:
```
"https://api.anyblock.tools/market/AMPL_USD_via_ALL/daily-volume?roundDay=false&debug=false&access_token=$ANYBLOCK_KEY&start=$EOD&end=$NOW",
```

Comment (krasi-georgiev) [reply configs/index.json:107]: yes this was a bug, good catch

**Comment by yuanaichi** [comment]
rapidapi  response the error message "This endpoint is disabled for your subscription"

**Comment by themandalore** [comment]
@imaginebeingatcomputers , any instructions for @yuanaichi ?

**Pull request merged**